### DISCUSSION TITLE
ADMIN_TOKEN with secure Argon2 PHC in LDAP configuration

### DISCUSSION THREAD

**Discussion by palicepta** [dani-garcia/vaultwarden#3762] [Q&A]
Dear All,

Acording to this article, I can use hashed password for Admin token:
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page#secure-the-admin_token

I don't have any problems with this.
But, how I can use hashed password (admin_token) in the LDAP synchronization config?

Thank you in advance.

**Comment by BlackDex** [comment]
You can't. You need to provide the plain/text password to any service or client which wants to connect to the admin interface.

**Comment by palicepta** [comment]
Thank you for your fast answer!
